docs/docker-in-30-mins.md

```diff
@@ -0,0 +1,127 @@
+# Docker in 30 mins with a Node.JS application #
+
+To start using docker for your application(s), you will need Docker installed on your machine and optionally a Dockerhub account to host your docker image.
+
+
+## Installing Docker ##
+
+Install docker. A list of the supported platforms and instructions for installing docker on those platforms are located [here](https://docs.docker.com/engine/installation/#supported-platforms).  
+
+Before you continue, ensure you can run the docker hello-world example by running `sudo docker run hello-world` in a terminal. You should see an output like the below:
+
+![hello-world-output](https://s24.postimg.org/vumoh4zw5/Selection_103.png)
+
+
+## Containerising your app ##
+
+### DockerFile ###
+
+To containerise your app you will need a DockerFile for your app. Create a file named 'DockerFile' in the root of your app directory with the following content:
+
+> FROM node:6.10.2-alpine
+> 
+> \# Create app directory  
+> RUN mkdir -p /usr/src/app  
+> WORKDIR /usr/src/app  
+> 
+> \# Install app dependencies  
+> COPY package.json /usr/src/app/  
+> RUN npm install  
+> 
+> \# Bundle app source  
+> COPY . /usr/src/app  
+> 
+> EXPOSE 8000  
+> 
+> CMD [ "npm", "start" ] # make sure start script in package.json
+
+** the above presumes your app listens on port 8000, and that you have a start script in your package.json:
+
+> "scripts": {  
+> &nbsp;&nbsp;&nbsp;&nbsp;"start": "node app.js"  
+>  },
+
+
+## Build your app docker container ##
+
+Run the following from the root of your app directory:
+
+* `docker build -t <your-username>/<your-app-name> .`
+
+If you have any errors regaring the location of your DockerFile, you may need to specify the path to your DockerFile:
+
+* `docker build -t <your-username>/<your-app-name> -f <location-and-path-of-dockerfile> .`
+
+You should see an output like the following:
+
+![build-output](https://s24.postimg.org/eimbvp6et/Selection_104.png)
```

Comment:
We should probably include the links in a docs folder instead of linking externally.